doc/ReleaseNotes

```diff
@@ -27,17 +19,40 @@ TCPDirect distributed as source code
   suitable new build and installation workflows for the TCPDirect source
   package.
 
-  There is no longer a 'tarball' distribution type as the RPM and DEB source
-  packages are suitable for all supported OSs.
+  In particular, this means an additional Onload development package
+  is required, which contains the necessary headers for building TCPDirect.
+
+
+New onload development package
+------------------------------
+
+  Onload now includes a package containing headers required by ef_vi
+  applications like TCPDirect. Installing this package is required to
+  build TCPDirect and TCPDirect applications.
+
+  Please ensure the openonload-devel RPM or onload-dev DEB is installed
+  after building Onload before attempting to build TCPDirect.
+
+
+Public Onload/ef_vi control plane API
+-------------------------------------
+
+  The Onload control plane in this Onload-9.0.0 is presented via a new
+  public API that can be used by ef_vi applications. As an ef_vi application,
+  TCPDirect now uses this API to query the control plane server.
```

Comment:
Can we swap the order of these two sections? If a customer gets stuck on the missing `onload-dev` message when installing, their first thought may be "_Did I miss a step?_" so would be best if its immediately before the installation section imo.

Reply:
I already switched it to this way round to have the new package section flow naturally from

> The packaging for Onload and TCPDirect has been refreshed to allow suitable new build and installation workflows for the TCPDirect source package."

Do you really think the idea hasn't been called out enough by the time they reach the installation instructions and the very verbose point 0?

Reply:
Its more because it answers the question of where a customer gets onload-dev from:
>  Please ensure the openonload-devel RPM or onload-dev DEB is installed
  **after building Onload** before attempting to build TCPDirect.

This sentence says you get it from building Onload (rather than some other source).

Reply:
I think we are in the margins of extra utility here. The main obstacle to your initial experience was that we had failed to generate the release notes adjacent to the package you trialed!

Reply:
Thats likely true. If you are happy with the current order, then that is fine I think.

Reply:
Thanks Daniel!